cli/cfncluster/cfncluster.py

```diff
@@ -406,79 +368,81 @@ def instances(args):
 def status(args):
     stack = ('cfncluster-' + args.cluster_name)
     config = cfnconfig.CfnClusterConfig(args)
-    cfnconn = boto.cloudformation.connect_to_region(config.region,aws_access_key_id=config.aws_access_key_id,
-                                                    aws_secret_access_key=config.aws_secret_access_key)
+
+    cfn = boto3.client('cloudformation', region_name=config.region,
+                       aws_access_key_id=config.aws_access_key_id,
+                       aws_secret_access_key=config.aws_secret_access_key)
 
     try:
-        status = cfnconn.describe_stacks(stack)[0].stack_status
+        status = cfn.describe_stacks(StackName=stack).get("Stacks")[0].get('StackStatus')
         sys.stdout.write('\rStatus: %s' % status)
         sys.stdout.flush()
         if not args.nowait:
             while ((status != 'CREATE_COMPLETE') and (status != 'UPDATE_COMPLETE') and (status != 'UPDATE_ROLLBACK_COMPLETE')
                    and (status != 'ROLLBACK_COMPLETE') and (status != 'CREATE_FAILED') and (status != 'DELETE_FAILED')):
                 time.sleep(5)
-                status = cfnconn.describe_stacks(stack)[0].stack_status
-                events = cfnconn.describe_stack_events(stack)[0]
-                resource_status = ('Status: %s - %s' % (events.logical_resource_id, events.resource_status)).ljust(80)
+                status = cfn.describe_stacks(StackName=stack).get("Stacks")[0].get('StackStatus')
+                events = cfn.describe_stack_events(StackName=stack).get('StackEvents')[0]
+                resource_status = ('Status: %s - %s' % (events.get('LogicalResourceId'), events.get('ResourceStatus'))).ljust(80)
                 sys.stdout.write('\r%s' % resource_status)
                 sys.stdout.flush()
             sys.stdout.write('\rStatus: %s\n' % status)
             sys.stdout.flush()
             if ((status == 'CREATE_COMPLETE') or (status == 'UPDATE_COMPLETE')):
                 state = poll_master_server_state(stack, config)
                 if state == 'running':
-                    outputs = cfnconn.describe_stacks(stack)[0].outputs
+                    outputs = cfn.describe_stacks(StackName=stack).get("Stacks")[0].get('Outputs')
                     for output in outputs:
```

Comment:
if 'outputs' is none you will get an Exception
'NoneType' object is not iterable